tests/typecheck/fields/test_related.yml

```diff
@@ -928,3 +928,18 @@
 
                 class SalesMan(models.Model):
                     client = models.ManyToManyField(CustomUser)
+
+-   case: test_fails_if_app_label_is_unknown_in_relation_field
+    main: |
+        from installed.models import InstalledModel
+    installed_apps:
+      - installed
+    files:
+        -   path: installed/__init__.py
+        -   path: installed/models.py
+            content: |
+                from django.db import models
+                class InstalledModel(models.Model):
+                    non_installed = models.ForeignKey(  # E: Cannot find model 'not_installed.NonInstalledModel' referenced in field 'non_installed'
```

Comment:
Is it also possible to ignore this error using a `# type: ignore` comment?

Reply:
I tested it and indeed it works.

Reply:
I didn't think mypy supported suppressing any errors produced by a plugin (yet?)

Refs:
- https://github.com/typeddjango/django-stubs/issues/1023#issuecomment-1178853194
- https://github.com/python/mypy/issues/12987

Reply:
I thought the same. Yet the test passes successfully with the following diff:

```patch
diff --git a/tests/typecheck/fields/test_related.yml b/tests/typecheck/fields/test_related.yml
index 7e9758ae..cc863373 100644
--- a/tests/typecheck/fields/test_related.yml
+++ b/tests/typecheck/fields/test_related.yml
@@ -940,6 +940,6 @@
             content: |
                 from django.db import models
                 class InstalledModel(models.Model):
-                    non_installed = models.ForeignKey(  # E: Cannot find model 'not_installed.NonInstalledModel' referenced in field 'non_installed'
+                    non_installed = models.ForeignKey(  # type: ignore
                         "not_installed.NonInstalledModel", on_delete=models.CASCADE
                     )
```

I'm not complaining 😆

Reply:
How interesting, have to compare how this error is emitted versus the one that couldn't be ignored